adapters/silvermob/silvermobtest/supplemental/invalid-silvermob-ext-object.json

```diff
@@ -0,0 +1,29 @@
+{
+  "expectedMakeRequestsErrors": [
+    {
+      "value": "ext.bidder not provided",
+      "comparison": "literal"
+    }
+  ],
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
+        "id": "some-impression-id",
+        "tagid": "my-adcode",
+        "video": {
+          "mimes": ["video/mp4"],
+          "w": 640,
+          "h": 480,
+          "minduration": 120,
+          "maxduration": 150
+        },
+        "ext": "Awesome"
+      }
+    ],
+    "site": {
+      "page": "test.com"
+    }
+  },
+  "httpCalls": []
```

Comment:
Please add `expectedMakeRequestsErrors` here

Reply:
@silvermob looks like you missed this one. Could you please update it?

Reply:
It was in the top of JSON, moved down and removed httpCalls:[]

Reply:
Ah! sorry I missed it. Thanks!